rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,2237 @@
+      title:  OS Architecture
+      type: string
+      required: false
+    os_description:
+      description: Description of the os
+      title: OS Description
+      type: string
+      required: false
+    os_family:
+      description: Family of the os
+      title: OS Family
+      type: string
+      required: false
+    os_name:
+      description: Name of the os
+      title: OS Name
+      type: string
+      required: false
+    os_system_name:
+      description: Name of the system os
+      title: OS System Name
+      type: string
+      required: false
+    os_type:
+      description: Type of os
+      title: OS Type
+      type: string
+      required: false
+    os_vendor:
+      description: Vendor of the os
+      title: OS Vendor
+      type: string
+      required: false
+    riskScore:
+      description: Risk score (with criticality adjustments) of the asset
+      title: Risk Score
+      type: float
+      required: false
+    severe_vulnerabilities:
+      description: Number of sever vulns
+      title: Severe Vulnerabilities
+      type: integer
+      required: false
+    tags:
+      description: Asset tags
+      title: Tags
+      type: '[]asset_tag'
+      required: false
+    total_vulnerabilities:
+      description: Total number of vulns
+      title: Total Vulnerabilities
+      type: integer
+      required: false
+    source:
+      description: Source of the asset
+      title: Source
+      type: string
+      required: false
+    unique_identifiers:
+      description: Uniqure aspects of the asset
+      title: Unique Identifiers
+      type: '[]identifiers'
+      required: false
+    new:
+      description: unknown
+      title: New
+      type: '[]string'
+      required: false
+    remediated:
+      description: unknown
+      title: Remediates
+      type: '[]string'
+      required: false
+
+  
+connection:
+  credentials:
+    title: API Key
+    description: API key from account
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  region:
+    title: Region
+    description: the region in which the plugin will work
+    type: string
+    required: true
+    example: us
+  max_pages:
+    title: Max Pages
+    description: Max pages returned, default 100
+    type: integer
+    required: false
+    default: 100
+    example: 10
+  
+
+actions:
+  scan:
+    title: Scan
+    description: Start a scan on a site
+    input:
+      asset_ids:
+        title: Asset IDs
+        description: IDs of the assets to scan
+        type: "[]string"
+        example: ["cdc978de-f683-4178-a1d9-d5a94a114b87-default-asset-135"]
+        required: false
+      name:
+        title: Name
+        description: The name of the scan
+        type: string
+        example: "test cloud scan"
+        required: true
+      ip:
+        title: IP
+        description: Primary IPv4 or IPv6 address of the asset
+        type: string
```

Comment:
I'm thinking it might be best if we were to handle a list of IPs instead of a single IP?